The job **`dist-i586-gnu-i586-i686-musl`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/7420556990/job/20192179373) [(plain)](https://github.com/rust-lang-ci/rust/commit/8143a5797f1e903209c3aadfba34630e045576ce/checks/20192179373/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
failures:

---- [codegen] tests/codegen/personality_lifetimes.rs stdout ----

error: verification with 'FileCheck' failed
status: exit status: 1
command: "/checkout/obj/build/x86_64-unknown-linux-gnu/llvm/build/bin/FileCheck" "--input-file" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/codegen/personality_lifetimes/personality_lifetimes.ll" "/checkout/tests/codegen/personality_lifetimes.rs" "--allow-unused-prefixes" "--check-prefixes" "CHECK,NONMSVC" "--dump-input-context" "100"
--- stderr -------------------------------
/checkout/tests/codegen/personality_lifetimes.rs:27:12: error: CHECK: expected string not found in input
/checkout/tests/codegen/personality_lifetimes.rs:27:12: error: CHECK: expected string not found in input
 // CHECK: [[SLOT:%[0-9]+]] = alloca { ptr, i32, [1 x i32] }
           ^
/checkout/obj/build/x86_64-unknown-linux-gnu/test/codegen/personality_lifetimes/personality_lifetimes.ll:32:18: note: scanning from here
define void @test() unnamed_addr #0 personality ptr @rust_eh_personality {
                 ^
/checkout/obj/build/x86_64-unknown-linux-gnu/test/codegen/personality_lifetimes/personality_lifetimes.ll:34:2: note: possible intended match here
 %0 = alloca { ptr, i32 }, align 4

Input file: /checkout/obj/build/x86_64-unknown-linux-gnu/test/codegen/personality_lifetimes/personality_lifetimes.ll
Check file: /checkout/tests/codegen/personality_lifetimes.rs


-dump-input=help explains the following input dump.
Input was:
<<<<<<
<<<<<<
            1: ; ModuleID = 'personality_lifetimes.93fccd7b7f2f669f-cgu.0' 
            2: source_filename = "personality_lifetimes.93fccd7b7f2f669f-cgu.0" 
            3: target datalayout = "e-m:e-p:32:32-p270:32:32-p271:32:32-p272:64:64-f64:32:64-f80:32-n8:16:32-S128" 
            4: target triple = "i586-unknown-linux-gnu" 
            5:  
            6: %S = type {} 
            7:  
            8: ; core::ptr::drop_in_place<personality_lifetimes::S> 
            9: ; Function Attrs: uwtable 
           10: define internal void @"_ZN4core3ptr45drop_in_place$LT$personality_lifetimes..S$GT$17hdcbbe118ff7a026aE"(ptr noalias noundef nonnull align 1 %_1) unnamed_addr #0 { 
           11: start: 
           12: ; call <personality_lifetimes::S as core::ops::drop::Drop>::drop 
           13:  call void @"_ZN66_$LT$personality_lifetimes..S$u20$as$u20$core..ops..drop..Drop$GT$4drop17h9e8b1f5c5cc539e5E"(ptr noalias noundef nonnull align 1 %_1) 
           14:  ret void 
           15: } 
           16:  
           17: ; <personality_lifetimes::S as core::ops::drop::Drop>::drop 
           18: ; Function Attrs: noinline uwtable 
           19: define void @"_ZN66_$LT$personality_lifetimes..S$u20$as$u20$core..ops..drop..Drop$GT$4drop17h9e8b1f5c5cc539e5E"(ptr noalias noundef nonnull align 1 %self) unnamed_addr #1 { 
           21:  ret void 
           22: } 
           23:  
           23:  
           24: ; personality_lifetimes::might_unwind 
           25: ; Function Attrs: noinline uwtable 
           26: define internal void @_ZN21personality_lifetimes12might_unwind17h390e1199c797bcc8E() unnamed_addr #1 { 
           28:  ret void 
           29: } 
           30:  
           30:  
           31: ; Function Attrs: uwtable 
           32: define void @test() unnamed_addr #0 personality ptr @rust_eh_personality { 
check:27'0                      X~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ error: no match found
           33: start: 
check:27'0     ~~~~~~~
           34:  %0 = alloca { ptr, i32 }, align 4 
check:27'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
check:27'1      ?                                  possible intended match
           35:  %_3 = alloca %S, align 1 
check:27'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~
           36:  %_1 = alloca %S, align 1 
check:27'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~
           37: ; invoke personality_lifetimes::might_unwind 
check:27'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           38:  invoke void @_ZN21personality_lifetimes12might_unwind17h390e1199c797bcc8E() 
check:27'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           39:  to label %bb1 unwind label %cleanup 
check:27'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           40:  
check:27'0     ~
           41: bb6: ; preds = %bb5, %cleanup 
check:27'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           42: ; invoke core::ptr::drop_in_place<personality_lifetimes::S> 
check:27'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           43:  invoke void @"_ZN4core3ptr45drop_in_place$LT$personality_lifetimes..S$GT$17hdcbbe118ff7a026aE"(ptr noalias noundef nonnull align 1 %_1) #4 
check:27'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           44:  to label %bb7 unwind label %terminate 
check:27'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           45:  
check:27'0     ~
           46: cleanup: ; preds = %bb2, %start 
check:27'0     ~~~~~~~~
           47:  %1 = landingpad { ptr, i32 } 
           48:  cleanup 
           49:  %2 = extractvalue { ptr, i32 } %1, 0 
           50:  %3 = extractvalue { ptr, i32 } %1, 1 
           51:  call void @llvm.lifetime.start.p0(i64 8, ptr %0) 
           52:  store ptr %2, ptr %0, align 4 
           53:  %4 = getelementptr inbounds i8, ptr %0, i32 4 
           54:  store i32 %3, ptr %4, align 4 
           55:  br label %bb6 
           56:  
           57: bb1: ; preds = %start 
           58: ; invoke personality_lifetimes::might_unwind 
           59:  invoke void @_ZN21personality_lifetimes12might_unwind17h390e1199c797bcc8E() 
           60:  to label %bb2 unwind label %cleanup1 
           61:  
           62: bb5: ; preds = %cleanup1 
           63: ; invoke core::ptr::drop_in_place<personality_lifetimes::S> 
           64:  invoke void @"_ZN4core3ptr45drop_in_place$LT$personality_lifetimes..S$GT$17hdcbbe118ff7a026aE"(ptr noalias noundef nonnull align 1 %_3) #4 
           65:  to label %bb6 unwind label %terminate 
           66:  
           67: cleanup1: ; preds = %bb1 
           68:  %5 = landingpad { ptr, i32 } 
           69:  cleanup 
           70:  %6 = extractvalue { ptr, i32 } %5, 0 
           71:  %7 = extractvalue { ptr, i32 } %5, 1 
           72:  call void @llvm.lifetime.start.p0(i64 8, ptr %0) 
           73:  store ptr %6, ptr %0, align 4 
           74:  %8 = getelementptr inbounds i8, ptr %0, i32 4 
           75:  store i32 %7, ptr %8, align 4 
           76:  br label %bb5 
           77:  
           78: bb2: ; preds = %bb1 
           79: ; invoke core::ptr::drop_in_place<personality_lifetimes::S> 
           80:  invoke void @"_ZN4core3ptr45drop_in_place$LT$personality_lifetimes..S$GT$17hdcbbe118ff7a026aE"(ptr noalias noundef nonnull align 1 %_3) 
           81:  to label %bb3 unwind label %cleanup 
           82:  
           83: bb3: ; preds = %bb2 
           84: ; call core::ptr::drop_in_place<personality_lifetimes::S> 
           85:  call void @"_ZN4core3ptr45drop_in_place$LT$personality_lifetimes..S$GT$17hdcbbe118ff7a026aE"(ptr noalias noundef nonnull align 1 %_1) 
           86:  ret void 
           87:  
           88: terminate: ; preds = %bb6, %bb5 
           89:  %9 = landingpad { ptr, i32 } 
           90:  filter [0 x ptr] zeroinitializer 
           91:  %10 = extractvalue { ptr, i32 } %9, 0 
           92:  %11 = extractvalue { ptr, i32 } %9, 1 
           93: ; call core::panicking::panic_in_cleanup 
           94:  call void @_ZN4core9panicking16panic_in_cleanup17hc7b506799e523aedE() #5 
           95:  unreachable 
           96:  
           97: bb7: ; preds = %bb6 
           98:  %12 = load ptr, ptr %0, align 4, !noundef !2 
           99:  %13 = getelementptr inbounds i8, ptr %0, i32 4 
          100:  %14 = load i32, ptr %13, align 4, !noundef !2 
          101:  call void @llvm.lifetime.end.p0(i64 8, ptr %0) 
          102:  %15 = insertvalue { ptr, i32 } poison, ptr %12, 0 
          103:  %16 = insertvalue { ptr, i32 } %15, i32 %14, 1 
          104:  resume { ptr, i32 } %16 
          105: } 
          106:  
          107: ; Function Attrs: uwtable 
          108: declare noundef i32 @rust_eh_personality(i32 noundef, i32 noundef, i64 noundef, ptr noundef, ptr noundef) unnamed_addr #0 
          110: ; core::panicking::panic_in_cleanup 
          110: ; core::panicking::panic_in_cleanup 
          111: ; Function Attrs: cold noinline noreturn nounwind uwtable 
          112: declare void @_ZN4core9panicking16panic_in_cleanup17hc7b506799e523aedE() unnamed_addr #2 
          113:  
          114: ; Function Attrs: nocallback nofree nosync nounwind willreturn memory(argmem: readwrite) 
          115: declare void @llvm.lifetime.start.p0(i64 immarg, ptr nocapture) #3 
          116:  
          117: ; Function Attrs: nocallback nofree nosync nounwind willreturn memory(argmem: readwrite) 
          118: declare void @llvm.lifetime.end.p0(i64 immarg, ptr nocapture) #3 
          119:  
          120: attributes #0 = { uwtable "probe-stack"="inline-asm" "target-cpu"="pentium" } 
          121: attributes #1 = { noinline uwtable "probe-stack"="inline-asm" "target-cpu"="pentium" } 
          122: attributes #2 = { cold noinline noreturn nounwind uwtable "probe-stack"="inline-asm" "target-cpu"="pentium" } 
          123: attributes #3 = { nocallback nofree nosync nounwind willreturn memory(argmem: readwrite) } 
          124: attributes #4 = { cold } 
          125: attributes #5 = { cold noreturn nounwind } 
          126:  
          127: !llvm.module.flags = !{!0} 
          128: !llvm.ident = !{!1} 
          129:  
          130: !0 = !{i32 8, !"PIC Level", i32 2} 
          131: !1 = !{!"rustc version 1.77.0-nightly (8143a5797 2024-01-05)"} 
          132: !2 = !{} 
------------------------------------------



```

</details>

--- a/tests/codegen/personality_lifetimes.rs
+++ b/tests/codegen/personality_lifetimes.rs
@@ -24,7 +24,7 @@ pub fn test() {
     let _s = S;
     // Check that the personality slot alloca gets a lifetime start in each cleanup block, not just
     // in the first one.
-    // CHECK: [[SLOT:%[0-9]+]] = alloca { ptr, i32, [1 x i32] }
+    // CHECK: [[SLOT:%[0-9]+]] = alloca { ptr, i32{{.*}} }
     // CHECK-LABEL: cleanup:
     // CHECK: call void @llvm.lifetime.start.{{.*}}({{.*}})
     // CHECK-LABEL: cleanup1:

